script/sendmail.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+#for saving the mail temporarily
+random=$RANDOM
+
+#Mail related functionality
+mail=$(which mailutils)
+mail_status=$?
+check_ser=$(which postfix)
+check_ser_status=$?
+temp_d=/tmp/
+mail_ext=.mail
+
+#Checking GUI related functions
+gui_check="$(startx >/dev/null 2>&1)"
```

Comment:
Servers may have X configured too. I guess no need to require that no X on the server to work for this script.

Reply:
It is merely a suggestion

Reply:
It had some side effects. I starts a new X session.

I suggest no checking of gui at all here.

Reply:
That would actually make a little sense. How about we just comment the code out, until we devise a better plan?